### PR TITLE
refactoring and commenting 

### PR DESCRIPTION
**Pull request by Frydafly**

### CODE REVIEW

**Comment by Frydafly** [comment]
@ryantimjohn sorry for the huge PR, it's a branch I've been updating for a bit. none of the changes are necessary, they just refactor or comment. no rush to review, and feel free to decline any of the changes.

**Comment by ryantimjohn** [comment]
Woah, @Frydafly, this is sick! Mind jumping on a call with me sometime next week to walk me through the changes? Put something on my cal!

File: utils/add_dependencies.py [hunk 1]
```diff
@@ -1,39 +1,64 @@
-#!/usr/bin/env python3
-# coding: utf-8
-
```
Comment (Frydafly) [inline utils/add_dependencies.py:3]: Add #TODO to add_dependencies for no version check

File: utils/create_or_update_standard_models.py [hunk 2]
```diff
@@ -47,8 +47,13 @@ def get_destination():
     print("This is usually the 'models' folder of the target repo.")
     print("This script will not let you create models in the current repo.")
     destination_path = ''
-    while not destination_path and not (destination_path.endswith(path.join('..', 'models'))
```
Comment (Frydafly) [inline utils/create_or_update_standard_models.py:50]: #TODO add inline comments to clarify

#TODO object oriented rewrite with classes :)

File: utils/create_or_update_standard_models.py [hunk 3]
```diff
@@ -258,9 +264,11 @@ def create_or_update_docs(docs_path, destination_path):
         schema_dict['models'].append(docs_model)
         print(
             f"\nCan we add docs for the current model to your schema.yml in {destination_path}?")
-        print("This is the data we'd add to your schema.yml:")
-        print(docs_model)
-        print("\nThe resultant yaml would look like this:")
+        _extracted_from_create_or_update_docs_36(
```
Comment (Frydafly) [inline utils/create_or_update_standard_models.py:267]: CONFUSING, return to three printing, or print all three...

File: utils/create_or_update_standard_models.py [hunk 4]
```diff
@@ -269,15 +277,15 @@ def create_or_update_docs(docs_path, destination_path):
...
-        return
-    else:
+    if docs_model != schema_dict['models'][existing_model_index]:
         docs_model_set = set(docs_model)
         schema_dict_model_set = set(
             schema_dict['models'][existing_model_index])
-        print("Your existing in the schema file for this model")
-        print("is different than the one in our documentation.")
-        print("Here's what exists in our documentation but not your schema file:")
+        _extracted_from_create_or_update_docs_36(
```
Comment (Frydafly) [inline utils/create_or_update_standard_models.py:284]: three prints.

Comment (Frydafly) [reply utils/create_or_update_standard_models.py:284]: print("line1\n"
         "line2\n"
        "line3")

File: utils/create_or_update_standard_models.py [hunk 5]
```diff
@@ -300,7 +315,7 @@ def delete_non_standard_model_choice(destination_file_path):
     """
     print('---------')
     with open(destination_file_path, 'r') as f:
-        for line in f.readlines():
+        for line in f:
```
Comment (Frydafly) [inline utils/create_or_update_standard_models.py:318]: `f.readlines()` add back

Comment (Frydafly) [reply utils/create_or_update_standard_models.py:318]: TEST output again--

**Review by ryantimjohn — APPROVED** [review]
LGTM when you make the changes you pointed out on code review.

File: utils/add_dependencies.py (hunk 1 above)
Comment (Frydafly) [reply utils/add_dependencies.py:3]: done

File: utils/create_or_update_standard_models.py (hunk 2 above)
Comment (Frydafly) [reply utils/create_or_update_standard_models.py:50]: done

File: utils/create_or_update_standard_models.py (hunk 3 above)
Comment (Frydafly) [reply utils/create_or_update_standard_models.py:267]: done

File: utils/create_or_update_standard_models.py (hunk 4 above)
Comment (Frydafly) [reply utils/create_or_update_standard_models.py:284]: done

File: utils/create_or_update_standard_models.py (hunk 5 above)
Comment (Frydafly) [reply utils/create_or_update_standard_models.py]: done

**Comment by Frydafly** [comment]
@ryantimjohn made the changes we discussed this morning!

**Comment by Frydafly** [comment]
used this version of the project on fairfight updates and it all looks good! gonna merge!!!! wooo

**Pull request merged**